ovos_audio/service.py

```diff
@@ -290,6 +294,10 @@ def _maybe_reload_tts(self):
         if config.get("module", "") == config.get("fallback_module", ""):
             return
 
+        if not config.get('preload_fallback', True):
```

Comment:
i would default this to False in ovos-config default mycroft.conf later, just leaving a note here

if default TTS is offline this should not be preloaded

Reply:
I just set this for backwards-compat; I'm setting it to False in Neon